gamebot/TicTacToe.cs

```diff
@@ -65,24 +65,24 @@ public string DrawGame()
         {
             int width = game.GetLength(0); //get the width of the game
             int height = game.GetLength(1); //get the height of the game
-            string result = " "; //create empty result with one space
+            string result = "▪️"; //create empty result with one space
 
             for (int i = 0; i < width; i++)
             {
-                result += i + 1;    //write 1 to whatever the width is
+                result += (i + 1) + "⃣";    //write 1 to whatever the width is
             }                   //example: width is 4, it writes 1234
             for (int i = 0; i < height; i++)
             {
                 result += '\n'; //everytime it go to the next row in the game, creates a new line
-                result += "ABCDEFGHIJKLMNOPQRSTUVWXYZ"[i];  //it also enters the current height line
+                result += ":regional_indicator_" + "abcdefghijklmnopqrstuvwxyz"[i] + ":";  //it also enters the current height line
```

Comment:
well, it uses emoji :one: :two: :three: :four: :five: :six: :seven: :eight: :nine: actually i'm not even sure if it works on mobile 🤔